src/components/organims/privacy/OffboardingComplete.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { StaticQuery, graphql, Link } from "gatsby";
+import { css } from 'emotion';
+import { rem } from 'polished';
+
+import { mqMin } from '../../../styles/breackpoint';
+import * as font from '../../../styles/fonts';
+
+import SectionMinimal from '../../molecules/SectionMinimal';
+import Title from '../../atoms/Title';
+import Button from '../../atoms/Button';
+
+class OffboardingComplete extends React.Component {
+  constructor(props) {
+    super(props);
+    this.logout = props.logout;
```

Comment:
This does not feel right. If `props` are changed (because of re-render..), it won't be up to date!